lib/plugins/aws/package/compile/events/cloudFront/index.test.js

```diff
@@ -1446,11 +1426,9 @@ describe('AwsCompileCloudFrontEvents', () => {
         awsCompileCloudFrontEvents.serverless.service.provider.compiledCloudFormationTemplate
           .Resources.CloudFrontDistribution.Properties.DistributionConfig.DefaultCacheBehavior
       ).to.eql({
+        CachePolicyId: '658327ea-f89d-4fab-a63d-7e88639e58f6',
         TargetOriginId: 's3/bucketname.s3.amazonaws.com',
         ViewerProtocolPolicy: 'allow-all',
-        ForwardedValues: {
-          QueryString: false,
-        },
         LambdaFunctionAssociations: [
           {
             EventType: 'viewer-request',
```

Comment:
It'll be nice to have some test (written with `runServerless`) that confirms on `cachePolicy` usage